workflow_test.go

```diff
@@ -0,0 +1,274 @@
+package workflow_test
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"flag"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/codingpop/workflow-trigger"
+)
+
+var owner = flag.String("owner", "", "Repository owner")
+var repo = flag.String("repo", "", "GitHub repository name")
+var accessToken = flag.String("access-token", "", "Personal access token")
+var eventType = flag.String("event-type", "", "A custom webhook event name")
+
+func init() {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+}
```

Comment:
Instead of all this, allow the http.Client to be configured in `workflow.Configure`, then use `server.Client()` to get a client that's happy to talk to your server.